Metrics.php

```diff
@@ -81,6 +82,20 @@ public static function getMetricTranslations()
         return $translations;
     }
 
+    public static function getMetricSemanticTypes()
+    {
+        $translations = [];
```

Comment:
I guess the variable name might be a bit missleading

Reply:
:+1: